### PR TITLE
Adds powerarmor (barely)!

### PR DESCRIPTION
**Pull request by Altoids1**
image

At the moment they don't have any special features, beyond maybe some high armor values.

Currently all this is dormant code, made for (and paid for by) Boodaliboo for a lavaland ruin they're intent on making.

#### Changelog

:cl:  Altoids
rscadd: Some sorts of Powerarmor have been added to the game, as dormant code you'll probably only ever find thru adminbus or a lavaland ruin.
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
yay

**Comment by MegaEmpirical** [comment]
Time to upgrade the death squad

**Comment by Ratvar** [comment]
That lavaland ruin is equivalent of adminbus is a big ehhhhh

**Comment by ghost** [comment]
>acid 0

File: yogstation/code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -3,3 +3,41 @@
...
+/obj/item/clothing/head/helmet/space/hardsuit/powerarmor
+	alternate_worn_icon = 'yogstation/icons/mob/head.dmi'
+	icon = 'yogstation/icons/obj/clothing/hats.dmi'	
+/obj/item/clothing/suit/space/hardsuit/powerarmor
+	alternate_worn_icon = 'yogstation/icons/mob/suit.dmi'
+	icon = 'yogstation/icons/obj/clothing/suits.dmi'
+
+/obj/item/clothing/head/helmet/space/hardsuit/powerarmor/t45b
+	name = "Salvaged T-45b helmet"
+	desc = "It's a pre-War power armor helmet, recovered and maintained by NCR engineers."
```
Comment (fluffe9911) [inline yogstation/code/modules/clothing/spacesuits/hardsuit.dm]: Fix these descriptions to be lore friendly

File: yogstation/code/modules/clothing/spacesuits/hardsuit.dm [hunk 2]
```diff
@@ -3,3 +3,41 @@
...
+	icon_state = "t45bhelmet"
+	item_state = "t45bhelmet"
+	armor = list("melee" = 50, "bullet" = 48, "laser" = 25, "energy" = 25, "bomb" = 48, "bio" = 100, "rad" = 50, "fire" = 50, "acid" = 0) 
+/obj/item/clothing/suit/space/hardsuit/powerarmor/t45b
+	name = "Salvaged T-45b power armor"
+	desc = "It's a set of T-45b power armor recovered by the NCR during the NCR-Brotherhood War."
```
Comment (fluffe9911) [inline yogstation/code/modules/clothing/spacesuits/hardsuit.dm]: same here

File: yogstation/code/modules/clothing/spacesuits/hardsuit.dm [hunk 3]
```diff
@@ -3,3 +3,41 @@
...
+	item_state = "t45bpowerarmor"
+	armor = list("melee" = 50, "bullet" = 48, "laser" = 25, "energy" = 25, "bomb" = 48, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/powerarmor/t45b
+
+/obj/item/clothing/head/helmet/space/hardsuit/powerarmor/advanced
+	name = "Advanced power helmet"
+	desc = "It's an advanced power armor Mk I helmet, typically used by the Enclave. It looks somewhat threatening."
+	icon_state = "advhelmet1"
+	item_state = "advhelmet1"
+	armor = list("melee" = 50, "bullet" = 48, "laser" = 25, "energy" = 25, "bomb" = 48, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
```
Comment (fluffe9911) [inline yogstation/code/modules/clothing/spacesuits/hardsuit.dm]: 0 acid resistance for a suit of power armor hmmm really makes you think also same with the other one

Comment (boodaliboo) [reply yogstation/code/modules/clothing/spacesuits/hardsuit.dm]: yeah they should acid resist

**Comment by ghost** [comment]
Hmm, maybe make the hardsuits lore from the UEG?

**Comment by Codeatmos** [comment]
https://www.youtube.com/watch?v=KwGtCsfKflk

**Pull request merged**